src/Microsoft.Health.Dicom.SqlServer/Microsoft.Health.Dicom.SqlServer.csproj

```diff
@@ -17,10 +17,11 @@
     <FrameworkReference Include="Microsoft.AspNetCore.App" />
     <PackageReference Include="Ensure.That" Version="9.2.0" />
     <PackageReference Include="Microsoft.Extensions.Diagnostics.HealthChecks" Version="3.1.8" />
-    <PackageReference Include="Microsoft.Health.Abstractions" Version="1.0.0-master-20200909-1" />
-    <PackageReference Include="Microsoft.Health.Extensions.BuildTimeCodeGenerator" Version="1.0.0-master-20200909-1" />
-    <PackageReference Include="Microsoft.Health.SqlServer" Version="1.0.0-master-20200909-1" />
-    <PackageReference Include="Microsoft.Health.SqlServer.Api" Version="1.0.0-master-20200909-1" />
+    <PackageReference Include="Microsoft.Health.Abstractions" Version="1.0.0-master-20201007-1" />
+    <PackageReference Include="Microsoft.Health.Extensions.BuildTimeCodeGenerator" Version="1.0.0-master-20201007-1" />
+    <PackageReference Include="Microsoft.Health.SqlServer" Version="1.0.0-master-20201007-1" />
+    <PackageReference Include="Microsoft.Health.SqlServer.Api" Version="1.0.0-master-20201007-1" />
+    <PackageReference Include="System.Data.SqlClient" Version="4.8.2" />
```

Comment:
Did we not move to Microsoft.Data.SqlClient?

Reply:
Yes, we did. However, for some reason, if I remove this package reference, I end up with these error messages:
_The type 'SqlDbType' exists in both 'System.Data.Common, Version=4.2.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' and 'System.Data.SqlClient, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'_

Reply:
I checked with Brandon. This is required to solve the mentioned issue caused because of dependency on fo-dicom. Fo-Dicom is referencing a very old version of System.Data.SqlClient. There are two possible solutions to it.
1. Updating the System.Data.SqlClient version in fo-dicom.
2. Switching from System.Data.SqlClient to Microsoft.Data.SqlClient.

Both of these solutions require fo-dicom to be updated. In the meantime, referencing the latest stable version of System.Data.SqlClient in Microsoft.Health.Dicom.SqlServer project seems to be the possible workaround.

Reply:
Thanks, Can you track a user story and update the fo-dicom?

Reply:
Absolutely, will do.